.github/workflows/testAndPublish.yml

```diff
@@ -430,6 +430,15 @@ jobs:
       run: ci/scripts/installNVDA.ps1
       env:
         nvdaLauncherDir: ${{ steps.getLauncher.outputs.download-path }}
+    - name: Download and Install Visual Studio Code
+      if: ${{ matrix.testSuite == 'vscode' }}
+      shell: pwsh
+      run: |
+        $ErrorActionPreference = 'Stop'
+        $setup = Join-Path $env:RUNNER_TEMP 'VSCodeSetup.exe'
+        Invoke-WebRequest -Uri 'https://code.visualstudio.com/sha/download?build=stable&os=win32-x64-user' -OutFile $setup
```

Comment:
Does `choco install vscode` fix installation issues?
https://community.chocolatey.org/packages/vscode

Reply:
Is choco available in the test environment? winget is inbox except on some images (LTSC, not sure re server).

Reply:
yes it is for windows runners, we use it elsewhere in this workflow

Reply:
Seems like it's still failing?

Reply:
It would be easier to read the logs if you log `ls dir` directly from GitHub actions after install vs code rather than in the system test